frontend/src/features/home/home.css

```diff
@@ -0,0 +1,29 @@
+header {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 10px;
+}
+
+.pools {
+    padding-left: 1em;
+    padding-right: 1em;
+}
+
+.pools-container {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    margin-top: 1em;
+}
+
+.sidepanel-bg {
+    background-color: white;
+    height: 100%;
+    padding: .5em;
+}
+
+header {
+    /* -webkit-box-shadow: 0px 0px 9px 3px rgba(41,41,41,.25);
```

Comment:
Remove commented lines

Reply:
lol it's not that deep homie. we not getting graded on the code